pyproject.toml

```diff
@@ -112,19 +93,17 @@ exclude_lines = [
   "if TYPE_CHECKING",
 ]
 
-[tool.black]
-target-version = ['py37']
-line-length = 100
-
 # https://github.com/charliermarsh/ruff
 [tool.ruff]
-target-version = "py37"
-line-length = 100
-show-source = true
-src = ["ConfigSpace", "test"]
+target-version = "py38"
+line-length = 88
```

Comment:
Nothing really, just that it's migrated to `ruff` and 88 is the default for both black and ruff so I figured it would be good to just keep it at its default value